activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -774,7 +774,7 @@ def rename_column(table_name, column_name, new_column_name)
       #
       # generates:
       #
-      #   CREATE INDEX by_branch_desc_party ON accounts(branch_id DESC, party_id ASC, surname)
+      #   CREATE INDEX index_accounts_on_branch_id_and_party_id_and_surname ON accounts(branch_id DESC, party_id ASC, surname)
```

Comment:
> An alternative fix which I didn't do would be to update the `add_index` code to have a `name` option passed with the value `by_branch_desc_party` and in that case we would keep the same index name in the SQL statement. Please let me know if this alternative is preferred.

Yeah, I'd prefer the alternative like as other above examples.

Reply:
Alright. I updated the PR to pass the `name` option to the `add_index` and keep the SQL statement for that sort order example.